libretro-common/file/file_path.c

```diff
@@ -650,6 +661,105 @@ bool path_is_absolute(const char *path)
    return false;
 }
 
+bool is_posix_path(const char* path)
+{
+   if (strchr(path, posix_path_delimiter))
+      return true;
+
+   return false;
+}
+
+bool is_windows_path(const char* path)
+{
+   if (strchr(path, posix_path_delimiter))
+      return false;
+
+   return true;
+}
+
```

Comment:
Hmm... These `is_posix_path()` and `is_windows_path()` functions feel a little wrong, since they are not properly generic. What I mean here is that `my_file` is a valid (relative) posix path, but `is_posix_path()` will return false in this case.

This `file_path.c` contains very important, low-level, abstracted code - so functions need to work properly in all cases :)

You could rename these to `path_contains_posix_delimiter()`/`path_contains_windows_delimiter()`, and then they would do what they say on the tin

(but we might not need the functions at all - see below)

Reply:
You are right about `my_file` not being handled correctly, I'll check this..

Reply:
Removed `is_posix_path()` and `is_windows_path()`.